tablo/interfaces/arcgis/views.py

```diff
@@ -212,13 +212,15 @@ def handle_request(self, request, **kwargs):
 
 
 class QueryView(FeatureLayerView):
-    query_limit_default = 50000
+    query_limit_default = 1000
```

Comment:
Can we set this to a higher value (like 10,000). 1,000 is pretty limiting on the UI, and I don't know that there's a need for it to be quite that limiting, especially since we're not allowing the related data.